Admin.js

```diff
@@ -11,7 +11,7 @@
 
 var Admin = {
 
-    collectionCounters: [],
+    lolCounters: [],
 
     /**
```

Comment:
- eslint: Unexpected var, use let or const instead.
- eslint: All 'var' declarations must be at the top of the function scope.
- eslint: Combine this with the previous 'var' statement.
- eslint: Strings must use doublequote.
- eslint: Strings must use doublequote.